maintenance/wikia/cronjobs/updateCurationCMSTopicRanks.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Swagger\Client\CurationCMS\Api\TopicsApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
+use \Wikia\Logger\WikiaLogger;
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_ALL);
+
+require_once(dirname(__FILE__) . '/../../Maintenance.php');
+
+class updateCurationCMSTopicRanks extends Maintenance {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class "updateCurationCMSTopicRanks" to match the regular expression ^[A-Z][a-zA-Z0-9]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS101)